src/docs/cookbook/forms/focus.md

```diff
@@ -12,28 +12,16 @@ js:
     url: https://dartpad.dev/inject_embed.dart.js
 ---
 
-When a text field is selected and accepting input,
-it is said to have "focus."
-Generally, users shift focus to a text field by tapping,
-and developers shift focus to a text field programmatically by
-using the tools described in this recipe.
+Khi một ô text được chọn và đang chờ được nhập vào gọi là trường text đó đang được focus (tập trung). Về cơ bản, người dùng sẽ focus vào một ô text bằng cách nhấn vào màn hình, còn lập trình viên sẽ focus một ô text bằng cách dùng các công cụ có trong bài hướng dẫn này.
```

Comment:
"Một trường text gọi là đang được focus (tập trung) khi nó được chọn và đang chờ được nhập vào"

Reply:
Frankly, both @trannamloc and @lightkeima 's text still not sufficiently clear to me 😅

Reply:
Frankly, both @trannamloc and @lightkeima 's text still not sufficiently clear to me 😅

Reply:
"được chọn và cho phép người dùng nhập vào"?, Should we add something like "ví dụ, người dùng nhấp vào ô text và bàn phím sẽ hiển thị để người dùng có thể nhập văn bản vào ô text đó." ?